sql/2022/accessibility/lighthouse_a11y_score.sql

```diff
@@ -0,0 +1,67 @@
+#standardSQL
+# Percentiles of lighthouse a11y score from 2019 - 2022
+SELECT
+  '2019_07_01' AS date,
+  percentile,
+  APPROX_QUANTILES(score, 1000)[OFFSET(percentile * 10)] AS score
+FROM (
+  SELECT
+    CAST(JSON_EXTRACT(report, '$.categories.accessibility.score') AS NUMERIC) AS score
+  FROM
+    `httparchive.lighthouse.2019_07_01_mobile`),
+  UNNEST([10, 25, 50, 75, 90]) AS percentile
+GROUP BY
+  date,
+  percentile
+
+UNION ALL
+
+SELECT
+  '2020_08_01' AS date,
+  percentile,
+  APPROX_QUANTILES(score, 1000)[OFFSET(percentile * 10)] AS score
+FROM (
+  SELECT
+    CAST(JSON_EXTRACT(report, '$.categories.accessibility.score') AS NUMERIC) AS score
+  FROM
+    `httparchive.lighthouse.2020_08_01_mobile`),
+  UNNEST([10, 25, 50, 75, 90]) AS percentile
+GROUP BY
+  date,
+  percentile
+
+UNION ALL
+
+SELECT
+  '2021_07_01' AS date,
+  percentile,
+  APPROX_QUANTILES(score, 1000)[OFFSET(percentile * 10)] AS score
+FROM (
+  SELECT
+    CAST(JSON_EXTRACT(report, '$.categories.accessibility.score') AS NUMERIC) AS score
+  FROM
+    `httparchive.lighthouse.2021_07_01_mobile`),
+  UNNEST([10, 25, 50, 75, 90]) AS percentile
+GROUP BY
+  date,
+  percentile
+
+UNION ALL
+
+SELECT
+  '2022_06_01' AS date,
+  percentile,
+  APPROX_QUANTILES(score, 1000)[OFFSET(percentile * 10)] AS score
+FROM (
+  SELECT
+    CAST(JSON_EXTRACT(report, '$.categories.accessibility.score') AS NUMERIC) AS score
+  FROM
+    `httparchive.lighthouse.2022_06_01_mobile`),
```

Comment:
Same request to add desktop for 2022.

Reply:
Adding desktop scores makes sense to me for all Lighthouse queries except this one, since it’s a comparison of YoY scores?

Reply:
Adding client will still allow you to do that YoY comparison for mobile and also have a 2022 desktop to mobile query.